assertj-core/src/test/java/org/assertj/core/api/classes/ClassAssert_isNotPrimitive_Test.java

```diff
@@ -0,0 +1,24 @@
+package org.assertj.core.api.classes;
```

Comment:
add the javadoc comment with the licence, this is why the build failed.
add it to every new files